### ISSUE TITLE
dock 'noVNC' control box on either top/bottom of the window rather than left/right only

### ISSUE THREAD

**Issue by yshen16** [novnc/noVNC#1471]
**Is your feature request related to a problem? Please describe.**
I was bothered by the extensive box the 'noVNC' control box shows when I try to copy/paste in my BU SCC OnDemand Desktop session. And the control box seems can only be put on either left or right side of the window. 

**Describe the solution you'd like**
I wonder if the following would be made possible: 
1. make the box can be put on top or bottom of the window
2. make the copy/paste text box resizable with scroll bar. 

**Describe alternatives you've considered**
No really

**Additional context**
I am attaching a screen shot for your reference. 
![newFeatureReq20201002-ys](https://user-images.githubusercontent.com/13019822/94956207-c4ac0200-04b9-11eb-892d-0bbca0539f49.PNG)

**Comment by samhed** [comment]
I agree, it would be nice if the control bar also could be placed on the top or the bottom.

Regarding the clipboard window, to summarize your ideas (from the screenshot as well):
 * Make the entire clipboard window resizable
 * Add history (perhaps in the form of tabs?)
 * Place the 'Clear' button on the top to save space

Scrollbars should appear if enough text is copied by the way.

**Comment by xiongchao902** [comment]
Clipboard and clipboardPasteFrom, How does this feature work without clipboard returning messages

**Comment by xiongchao902** [comment]
rfb.addEventListener("clipboard", clipboards);
function clipboards (e){
            console.log(e)
}
console.log not  work

**Comment by tobfah** [comment]
#2043
